src/bonbast/data.py

```diff
@@ -118,3 +118,51 @@ def to_json(self) -> dict:
                 'price': self.price,
             }
         }
+
+
+class Graph:
+    """ Graph model
+    """
+    VALUES = {
```

Comment:
These are the same as currencies. We don't want to have duplicates.
Use what we have in the currencies.